dfastmi/gui/dialog_view_model.py

```diff
@@ -179,10 +180,11 @@ def qthreshold(self, value: float):
         value = max(value, self.current_reach.qstagnant)
         self._qthreshold = value
         self.model.qthreshold = value
+        self._qthreshold_cache[(self.current_branch, self.current_reach)] = value
 
         self._update_slength()
         # Notify the view of the change
-        self.qthreshold_changed.emit(self._qthreshold)
+        self.qthreshold_changed.emit(self.qthreshold, self.current_reach.qstagnant)
```

Comment:
shouldn't qstagnant have its own property and emit if it is needed?

Reply:
no

Reply:
isn't it now just piggybacking on the qthreshold emit?

Reply:
qstagnant is the discharge stagnant of a reach (minimum/'drempel'), threshold can be higher.